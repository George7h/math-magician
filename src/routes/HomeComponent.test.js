// Home.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import Home from './HomeComponent'; // Import your component

test('Home component matches snapshot', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
