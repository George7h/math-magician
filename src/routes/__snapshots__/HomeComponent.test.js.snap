// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component matches snapshot 1`] = `
<div
  className="welcome"
>
  <h1>
    Welcome to our page
  </h1>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Eum, ea. Fugiat laborum ut, quasi voluptate odit nobis reiciendis maiores aut iure unde porro similique pariatur deleniti quisquam, ratione voluptatem aliquid. Lorem ipsum dolor sit amet consectetur adipisicing elit. Eum, ea. Fugiat laborum ut, quasi voluptate odit nobis reiciendis maiores aut iure unde porro similique pariatur deleniti quisquam, ratione voluptatem aliquid. Lorem ipsum dolor sit amet consectetur adipisicing elit. Eum, ea. Fugiat laborum ut, quasi voluptate odit nobis reiciendis maiores aut iure unde porro similique pariatur deleniti quisquam, ratione voluptatem aliquid.
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Eum, ea. Fugiat laborum ut, quasi voluptate odit nobis reiciendis maiores aut iure unde porro similique pariatur deleniti quisquam, ratione voluptatem aliquid.Lorem ipsum dolor sit amet consectetur adipisicing elit. Eum, ea. Fugiat laborum ut, quasi voluptate odit nobis reiciendis maiores aut iure unde porro similique pariatur deleniti quisquam, ratione voluptatem aliquid.Lorem ipsum dolor sit amet consectetur adipisicing elit. Eum, ea. Fugiat laborum ut, quasi voluptate odit nobis reiciendis maiores aut iure unde porro similique pariatur deleniti quisquam, ratione voluptatem aliquid.
  </p>
</div>
`;
